### PR TITLE
framework/media: Refine rechanneling logic in audio manager

### PR DESCRIPTION
**Pull request by zhouxinhe**
audio manager does not need to care rechanneling logic, which has been implemented in src_simple().

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [535995c2460dd4b5aa4595a437b708d5d1ee84a0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [535995c2460dd4b5aa4595a437b708d5d1ee84a0] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [51e6f5ea9c6f6030e67b47ec49ca49a819ac9fee] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [51e6f5ea9c6f6030e67b47ec49ca49a819ac9fee] - Code Rule Check OK.

File: framework/src/media/audio/audio_manager.c [hunk 1]
```diff
@@ -404,94 +405,103 @@ static uint32_t get_closest_samprate(unsigned int origin_samprate, audio_io_dire
...
-	if ((srcData.origin_channel_num = pcm_get_channels(card->pcm)) == 0) {
-		meddbg("Fail to get channel number\n");
-		return AUDIO_MANAGER_OPERATION_FAIL;
-	}
-
+	srcData.origin_channel_num = pcm_get_channels(card->pcm);
 	srcData.origin_sample_rate = pcm_get_rate(card->pcm);
-	srcData.origin_sample_width = SAMPLE_WIDTH_16BITS;
-	srcData.desired_channel_num = srcData.origin_channel_num; // FIXME: card->resample.user_channel
+	srcData.origin_sample_width = SAMPLE_WIDTH_16BITS; // pcm_get_format(card->pcm) == PCM_FORMAT_S16_LE
```
Comment (Taejun-Kwon) [inline framework/src/media/audio/audio_manager.c]: please remove old code here.

**Comment by seinfra** [comment]
Target : [52dcff3d06d06ede07f162e97c2c4fb3f2fbee24] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [52dcff3d06d06ede07f162e97c2c4fb3f2fbee24] - Code Rule Check (C++) OK.

File: framework/src/media/audio/audio_manager.c (hunk 1 above)
Comment (zhouxinhe) [reply framework/src/media/audio/audio_manager.c]: ok, I removed it.

**Comment by Taejun-Kwon** [comment]
Thanks for your work, looks good to me.

**Pull request merged**